kyliepost/cars-r-us
# fixed some bugs

# Fixed some bugs, got choices to pop up

## Steps to test
1. open code
2. open server
3. check to see that choices are popping up even tho they are undefined
